### PR TITLE
Fix-issue-Links

### PR DESCRIPTION
**Pull request by alejandrocabriales**
# Ticket
https://trello.com/b/iebIfrww/ses-dashboard

# Actions
Fix-link-login-screen

### CODE REVIEW

File: src/stories/components/footer/description-footer.tsx [hunk 1]
```diff
@@ -13,23 +13,18 @@ interface Props {
...
     <div style={style}>
       <StyleTitle isLight={isLight}>{title}</StyleTitle>
       {children &&
         children.map((item, index) =>
           item.isNotLink ? (
-            <CookiesLink isLight={isLight} onClick={HandleOnClick(item.url)} key={index}>
-              {item.title}
-            </CookiesLink>
+            <Link href={item.url} passHref legacyBehavior>
+              <CookiesLink isLight={isLight} key={index}>
```
Comment (YasielCabrera) [inline src/stories/components/footer/description-footer.tsx:24]: You should move up the key to avoid issues

**Review by YasielCabrera — CHANGES_REQUESTED** [review]

File: src/stories/components/footer/description-footer.tsx (hunk 1 above)
Comment (alejandrocabriales) [reply src/stories/components/footer/description-footer.tsx]: ok

Comment (alejandrocabriales) [reply src/stories/components/footer/description-footer.tsx]: You are right

**Review by YasielCabrera — APPROVED** [review]
LGTM

**Pull request merged**